lib/VoiceReadout.js

```diff
@@ -0,0 +1,17 @@
+require('../webOS/webOS');
+
+var
+	options = require('enyo/options');
+
+module.exports = {
+	/**
+	* Read alert message.
+	*
+	* @public
+	*/
+	readAlert: function(s) {
+		if (options.accessibility && window.webOS && window.webOS.voicereadout) {
```

Comment:
Basically readAlert api is only for accessibility. If we don't check option.accessiblity, it always read something out when calling realAlert even though the app sets option.accessibility false. I think this is weird. If developers want to use tts api they only use serviceRequest api.
For second question you're right, webOS checks is not necessary. I'll remove webOS check.